### PR TITLE
Add R to docker image and tests

### PR DESCRIPTION
**Pull request by susanxhuynh**
1. Added R to Docker image.
2. Added SparkR test, disabled for now. Test runs the "dataframe.R" example script from Spark examples.
@mgummelt

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -35,6 +35,8 @@ RUN apt-get update && \
 RUN add-apt-repository ppa:openjdk-r/ppa
 RUN apt-get update && \
     apt-get install -y openjdk-8-jdk curl
+RUN apt-get update && \
```
Comment (mgummelt) [inline docker/Dockerfile]: Remove update.  It's done in the step above.

**Comment by mgummelt** [comment]
docs?

File: tests/test.py [hunk 2]
```diff
@@ -96,6 +103,12 @@ def main():
         "Pi is roughly 3",
         py_file_path=py_file_path)
 
+    # TO DO: enable R test when R is enabled in Spark (2.1)
```
Comment (mgummelt) [inline tests/test.py]: remove space between "TO" and "DO".  People often search code bases for "TODO", and would miss this.

**Comment by susanxhuynh** [comment]
@mgummelt I made the changes  you requested.

Re: docs: I didn't want to update the docs until we actually support R, which will be enabled in Spark 2.1.

**Comment by mgummelt** [comment]
Ah, right.  LGTM.

**Pull request merged**